src/test/java/seedu/mark/testutil/ReminderBuilder.java

```diff
@@ -0,0 +1,68 @@
+package seedu.mark.testutil;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+import seedu.mark.model.bookmark.Bookmark;
+import seedu.mark.model.reminder.Note;
+import seedu.mark.model.reminder.Reminder;
+
+/**
+ * A utility class to help with building Reminder objects.
+ */
+public class ReminderBuilder {
+    public static final String DEFAULT_NOTE = "See Website";
+    public static final Bookmark DEFAULT_BOOKMARK = TypicalBookmarks.ALICE;
```

Comment:
Import the static variable `ALICE` from `TypicalBookmarks` first then you do not need to specify `TypicalBookmarks` here (just to be consistent with other tests)

Reply:
Okay